src/connectors/wdr.ts

```diff
@@ -0,0 +1,50 @@
+export {};
+
+// Connector for the WDR radio stations (1Live, WDR2, WDR3, WDR4, WDR5, COSMO, etc.)
+
+const filter = MetadataFilter.createFilter({
+	artist: cleanupArtist,
+});
+
+Connector.playerSelector = '.wdrrCurrentChannels';
+
+Connector.artistTrackSelector = '.wdrrCurrentShowTitleTitle';
+
+Connector.pauseButtonSelector = '#playCtrl.playing';
+
+Connector.isScrobblingAllowed = () => {
+	const artistTrack = Util.getTextFromSelectors(
+		Connector.artistTrackSelector,
+	);
+	const disallowedStrings = [
+		'1Live',
+		'WDR 2',
+		'WDR 3',
+		'WDR 4',
+		'WDR 5',
+		'COSMO',
+	];
+
+	// Check if the artistTrack includes any of the disallowed strings
+	const containsDisallowedString =
+		artistTrack !== null &&
+		disallowedStrings.some((disallowedString) =>
+			artistTrack.includes(disallowedString),
+		);
+
+	// Scrobble only if none of the disallowed strings are included
+	return !containsDisallowedString;
+};
+
+function cleanupArtist(artist: any) {
```

Comment:
```suggestion
function cleanupArtist(artist: string) {
```
I believe this will fix the linter errors